docs/config.yaml

```diff
@@ -60,6 +60,8 @@ streams:
       source_config_ref: main
       # OPTIONAL: override OS variant to use for this stream
       variant: rhcos-9.0
+      # OPTIONAL: require kernel + kernel-rt versions to match
+      check_kernel_rt_mismatch_rhcos: true
```

Comment:
Would we want this only for specific streams? Would we ever want it to apply globally to all streams?

Reply:
If we applied it to all streams, then we would have to disable the check for all streams just to turn this off for e.g. `4.11` only. I considered both options, but figured this gave us more granular control over the check. WDYT?

Reply:
I think having it be global would probably be fine. AIUI, the use case for temporarily disabling it is for hotfix builds. In that workflow, ART creates a separate pipecfg with the hotfix details, and can turn off the knob if necessary without affecting the prod streams. We can always make it stream-level in the future if needed.

Reply:
Alright, I can make that change. Where do we want that to live? Just a general `pipecfg.check_kernel_mismatch_rhcos` or add in another layer such as e.g. FCOS's `pipecfg.misc` layer?

Reply:
Under `misc` SGTM!